test/extended/OWNERS

```diff
@@ -6,7 +6,6 @@ reviewers:
   - gabemontero
   - kargakis
   - deads2k
-  - jim-minter
```

Comment:
if you remove yourself here, please add yourself to the build/template/imageecosystem subdirs.